quiz/src/test/java/rx/quiz/collections/BeginnerTerminalOperationsKoanTest.java

```diff
@@ -0,0 +1,17 @@
+package rx.quiz.collections;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public class BeginnerTerminalOperationsKoanTest extends BeginnerTerminalOperationsKoan {
+
+    @Override
+    public int sumOfEvenNumbers(@NotNull List<String> numbers) {
+        return numbers.stream()
+                .map(Integer::parseInt)
+                .filter(value -> value % 2 == 0)
+                .reduce(Integer::sum)
```

Comment:
Alternatively you can use reduce(0, Integer::sum)

Reply:
Yes, thanks. It's better this way.